### PR TITLE
WIP: add insufficient funds handling into bridge

### PR DESCRIPTION
**Pull request by ZacharyCouchman**
# Summary
Adding insufficient funds error handling back into Bridge Review screen.

Adjust amount button is only shown when the user is trying to move native and they have the ability to reduce the move amount so that they can afford move amount + gas.

**Trying to move native token with not enough for move + gas**
image

**Trying to move native from L1 with not enough native amount + gas**
image

### CODE REVIEW

**Review by sharonsheah — APPROVED** [review]
🚀

File: packages/checkout/widgets-lib/src/components/NotEnoughGas/NotEnoughGas.tsx [hunk 1]
```diff
@@ -58,86 +58,63 @@ NotEnoughGasProps) {
...
-          </Box>
-          <Box sx={logoContainerStyles}>
-            <Logo
-              testId="footer-logo-image"
-              logo="ImmutableHorizontalLockup"
-              sx={{ width: 'base.spacing.x25' }}
-            />
-          </Box>
         </Box>
+        <FooterLogo />
```
Comment (sharonsheah) [inline packages/checkout/widgets-lib/src/components/NotEnoughGas/NotEnoughGas.tsx:117]: Not sure if this is part of the design. For the drawers that I created, the figma designs didn't have the footer logo 🤔

File: packages/checkout/widgets-lib/src/components/NotEnoughGas/NotEnoughGas.tsx [hunk 2]
```diff
@@ -58,86 +58,63 @@ NotEnoughGasProps) {
...
-      visible={visible}
-      showHeaderBar={showHeaderBar}
-    >
-      <Drawer.Content>
-        <Box testId="not-enough-gas-bottom-sheet" sx={containerStyles}>
-          <CloudImage
-            imageUrl={
+    <Drawer size="threeQuarter" visible={visible} showHeaderBar={false}>
+      <Drawer.Content testId="not-enough-gas-bottom-sheet" sx={containerStyles}>
+        <CloudImage
```
Comment (sharonsheah) [inline packages/checkout/widgets-lib/src/components/NotEnoughGas/NotEnoughGas.tsx:63]: I think this needs some padding top and padding bottom to spread out the image from the heading + body text

Comment (ZacharyCouchman) [reply packages/checkout/widgets-lib/src/components/NotEnoughGas/NotEnoughGas.tsx:63]: Yep, I'll fix this up and confirm with Cass

File: packages/checkout/widgets-lib/src/components/NotEnoughGas/NotEnoughGas.tsx (hunk 1 above)
Comment (ZacharyCouchman) [reply packages/checkout/widgets-lib/src/components/NotEnoughGas/NotEnoughGas.tsx]: have removed it now

**Review by sharonsheah — APPROVED** [review]

**Review by dreamoftrees — APPROVED** [review]
👍 Awesome nice work @ZacharyCouchman

**Pull request merged**